enterprise/api/v1/serializers.py

```diff
@@ -463,6 +463,21 @@ def create(self, attrs):  # pylint: disable=arguments-differ
         return new_user, created
 
 
+class LinkLearnersSerializer(PendingEnterpriseCustomerUserSerializer):
+    """
+    Extends the PendingEnterpriseCustomerSerializer to validate that the enterprise customer uuid
+    matches the uuid the user has permissions to update
+    """
+
+    def validate_enterprise_customer(self, data):
+        """
+        Check that the enterprise customer is the same as the one the user has permissions for
+        """
+        if data['enterprise_customer'] != self.context.get('enterprise__uuid'):
```

Comment:
Do you need to handle the super case where the lack of a uuid means the user has permission to edit all? @iloveagent57 would have the expertise here.

Reply:
This method isn't defined on the parent case, so there's no need to call super here. DRF will run the regular validation and then call this method, is my understanding of it.

Reply:
If the user is a superuser or staff user, they can use the original endpoint that LicenseManager is using now; it doesn't make this check. That way they can use it for multiple enterprises.

Reply:
Got it, makes sense.